docs/collector.dhcp.md

```diff
@@ -0,0 +1,54 @@
+# dhcp collector
+
+The dhcp collector exposes DHCP Server metrics
+
+|||
+-|-
+Metric name prefix  | `dhcp`
+Data source         | Perflib
+Classes             | `DHCP Server`
+Enabled by default? | No
+
+## Flags
+
+None
+
+## Metrics
+
+Name | Description | Type | Labels
+-----|-------------|------|-------
+`packets_received_total` | Total number of packets received by the DHCP server | counter | None
+`duplicates_dropped_total` | Total number of duplicate packets received by the DHCP server | counter | None
+`packets_expired_total` | Total number of packets expired in the DHCP server message queue | counter | None
+`milliseconds_per_packet_avg` | Average time per packet taken by the DHCP server to send a response | gauge | None
```

Comment:
This doesn't actually seem to be exposed?

Reply:
These are exposed as far as I can tell (metrics are present).

Milliseconds per packet avg metric was dropped in a previous push.

Reply:
Yes, but it still seems to be in the docs in the latest version?

Reply:
Apologies for missing that Calle! I've removed it from the docs.